projects/dsp-ui/src/lib/core/session.service.ts

```diff
@@ -0,0 +1,215 @@
+import { Inject, Injectable } from '@angular/core';
+import {
+    ApiResponseData,
+    ApiResponseError,
+    Constants,
+    CredentialsResponse,
+    KnoraApiConfig,
+    KnoraApiConnection,
+    UserResponse
+} from '@dasch-swiss/dsp-js';
+import { DspApiConfigToken, DspApiConnectionToken } from './core.module';
+
+/**
+ * Currently logged-in user information
```

Comment:
Information about the current user?

Reply:
Yes

Reply:
Could you change the comment accordingly to `// information about the current user`?